docs/pages/database-access/guides/postgres-self-hosted.mdx

```diff
@@ -274,18 +200,21 @@ when connecting to the database instance:
 $ tsh db login --db-user=postgres --db-name=postgres example
 ```
 
-When logging into a PostgreSQL database, `tsh` automatically configures a section
-in the [connection service file](https://www.postgresql.org/docs/current/libpq-pgservice.html)
-with the name of `<cluster-name>-<database-service-name>`.
+Once logged in, connect to the database:
```

Comment:
Overall, the steps are great. I love the brevity. You use one or two sentences, then a command.

Reply:
Yeah I like how the new `tsh db connect` flow turned out, simpler than it was before and more uniform.